src/Selda/PG/Class.purs

```diff
@@ -3,27 +3,38 @@ module Selda.PG.Class
   , class InsertRecordIntoTableReturning
   , insertRecordIntoTableReturning
   , insert_
+  , insert_'
   , insert
+  , insert'
   , insert1
+  , insert1'
   , insert1_
+  , insert1_'
   , query
+  , query'
   , query1
+  , query1'
```

Comment:
The appended `'` indicates value-level codecs are used. I think a different suffix should be used that's clearer (e.g. `V` for value).